examples/chip-tool/commands/common/Command.cpp

```diff
@@ -237,7 +288,18 @@ size_t Command::AddArgument(const char * name, const char * value)
 size_t Command::AddArgument(const char * name, char ** value)
 {
     Argument arg;
-    arg.type  = ArgumentType::String;
+    arg.type  = ArgumentType::CharString;
+    arg.name  = name;
+    arg.value = reinterpret_cast<void *>(value);
```

Comment:
Surprised why we don't pass down a std::vector<char> here with a copy of the value from string, so that the later bits don't need to deal with null-termination.

Reply:
We could.  We could even store this as an std::string.  I was trying to not scope-creep this too much....